pages/about.html

```diff
@@ -38,3 +26,92 @@
+        margin-bottom: 15px;
+        padding: 0 10px;
+        text-align: center
+    }
+    #content .team > li > img {
+        margin-bottom: 15px;
+        width: 90px;
+    }
+    #content .team > li > h3 > span {
+        display: block;
+        font-size: 0.8125em;
+        margin-top: 10px;
+    }
+    #content .team > li > p {
+        text-align: left;
+    }
+    
+    @media (min-width:700px) {
+        #content .team > li {
+            padding-left: 0;
+            text-align: left;
+        }
+        #content .team > li:after {
+            clear: both;
+            content: " ";
+            display: block;
+        }
+        #content .team > li > img {
+            float: left;
+            margin: 0 20px 15px 0;
+            width: 120px;
+        }
+        #content .team > li > h3 > span {
+            display: inline-block;
+            font-size: 1em;
+            margin-top: 0;
+        }
+        #content .team > li > h3 > span:before {
+            content: "\00a0\2014\00a0";
+        }
+        #content .team > li > p {
+            max-width: 800px;
+        }
+    }
+</style>
+
+<ul class="team">
+    <li>
+        <img src="https://secure.gravatar.com/avatar/c7ebb4bf8014513304378a9a4d025faa?size=120" alt="Avatar Alexander Schmitz">
+        <h3><a href="https://github.com/arschmitz" title="Alexander Schmitz profile">Alexander Schmitz</a><span>Project Lead</span></h3>
+        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed luctus a orci id luctus. Praesent sed ipsum nisl. Pellentesque eu orci id neque efficitur tempus. Ut feugiat ornare magna, ut consequat felis rutrum vitae.</p>
+    </li>
+    <li>
+        <img src="https://secure.gravatar.com/avatar/b2bef16d251e5dfe3be5e31182a822b6?size=120" alt="Avatar Anne-Gaelle Colom">
+        <h3><a href="https://github.com/agcolom" title="Anne-Gaelle Colom GitHub profile">Anne-Gaelle Colom</a><span>Documentation Lead</span></h3>
+        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed luctus a orci id luctus. Praesent sed ipsum nisl. Pellentesque eu orci id neque efficitur tempus. Ut feugiat ornare magna, ut consequat felis rutrum vitae.</p>
+    </li>
+    <li>
+        <img src="https://secure.gravatar.com/avatar/acabeff49b5ba95a8e94de9445175f56?size=120" alt="Avatar Gabriel Schulhof">
+        <h3><a href="https://github.com/gabrielschulhof" title="Gabriel Schulhof GitHub profile">Gabriel Schulhof</a></h3>
+        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed luctus a orci id luctus. Praesent sed ipsum nisl. Pellentesque eu orci id neque efficitur tempus. Ut feugiat ornare magna, ut consequat felis rutrum vitae.</p>
+    </li>
+    <li>
+        <img src="https://secure.gravatar.com/avatar/691460b4005b0c16bcb96dcdf1f0e02d?size=120" alt="Avatar Cory Gackenheimer">
+        <h3><a href="https://github.com/cgack" title="Cory Gackenheime GitHub profile">Cory Gackenheimer</a></h3>
```

Comment:
Missing the last letter of my last name in the `title` attribute in the link to my github profile